### DISCUSSION TITLE
Draw circle of a fixed radius

### DISCUSSION THREAD

**Discussion by rezw4n** [geoman-io/leaflet-geoman#1547] [Ideas]
Hello. I know I can draw a circle but what if I want my circle to be exactly 2.5m long?
Can a feature be implemented while drawing a circle I can press 'r' and type in the radius of the circle like Arcmap/ArcGIS Pro?

**Comment by Falke-Design** [comment]
You can add the shortcut and the input into your own GUI and then setting the radius to a fixed size:

```
map.pm.setGlobalOptions({
	resizeableCircle: false,
	pathOptions: {radius: 4000},
	templineStyle: {radius: 4000},
})
```
https://plnkr.co/edit/ei5AFsLW8npdaxhs
